airflow/providers/google/cloud/operators/mlengine.py

```diff
@@ -1057,6 +1069,9 @@ def execute(self, context):
             self.log.info('MLEngine Training job request is: %s', training_request)
             return
 
+        if self._labels:
+            training_request['trainingInput']['labels'] = self._labels
```

Comment:
I have the impression that the labels should be in ``training_request`, not ``trainingInput``. From what I can see, the API reference documentation for this product is incomplete and the description for the training_input field is missing, 
https://cloud.google.com/ai-platform/prediction/docs/reference/rest/v1/projects.jobs#Job
so you have to look at the Discovery documentation which gets updated more frequently so you have to look at the Discovery documentation, which gets more frequent updates, but also has bugs and missing parameter descriptions.
https://ml.googleapis.com/$discovery/rest?version=v1
More info about Discovery docs: https://developers.google.com/discovery

<img width="773" alt="Screenshot 2020-08-07 at 17 24 00" src="https://user-images.githubusercontent.com/12058428/89661304-cd75c280-d8d2-11ea-940a-98c9c1bf04f3.png">

Reply:
Thanks for the links. I will take a look and come back with some changes.